theurgy.lic

```diff
@@ -62,6 +62,66 @@ def clean_altar
   put_in_container($settings.water_holder)
 end
 
+def prayer_bead
+  return if DRSkill.getrank('Mechanical Lore') < 140
+
+  fput "stow #{GameObj.right_hand.noun}" if GameObj.right_hand.name != 'Empty'
+  fput "stow #{GameObj.left_hand.noun}" if GameObj.left_hand.name != 'Empty'
+  get_from_container('prayer chain')
```

Comment:
I actually checked this against $settings.aspect instead.  That way if you want to carve beads without having commune truffinyi you can still do it to at least fill a chain - and will likely add a similar call while at the altar later in Theurgy.lic to meditate on the chain so you get the devotion boost.